### DISCUSSION TITLE
<Connecting GitHub to Quick Access>

### DISCUSSION THREAD

**Discussion by NotSomeBlox** [community/community#127457] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

I'm trying to connect GitHub to File Explorer and I don't know how to do it and add it to Quick Access

**Comment by karanntech** [comment]
To connect GitHub to File Explorer and add it to Quick Access, you can't directly link them like you would with a cloud storage service. However, you can achieve similar convenience by utilizing GitHub Desktop and File Explorer's.
1.Download GitHub Desktop: Go to the GitHub Desktop website and download the application. Install it on your computer.
2.Sign in to GitHub Desktop: Launch GitHub Desktop and sign in with your GitHub account.
3.Clone a Repository: In GitHub Desktop, clone a repository from your GitHub account to your local machine. This creates a local copy of the repository on your computer.
4.Open File Explorer: Navigate to the location where you cloned the repository.
5.Pin to Quick Access: Right-click on the folder representing your cloned repository in File Explorer. Then, select "Pin to Quick Access." This action adds a shortcut to the repository in the Quick Access section of File Explorer.

Hope this will help you :)
